.travis.yml

```diff
@@ -41,4 +40,4 @@ script:
 
 after_success:
 
-  - ./ci_support/upload_or_check_non_existence.py ./recipe conda-forge --channel=pre
```

Comment:
you are right, I didn't want to merge this, although I realised what I had done, killed all the builds and then updated to the 0.7 final release, which is why there is nothing wrong on anaconda.org. Thanks for the heads up.